presto-main/src/main/java/com/facebook/presto/metadata/SessionPropertyManager.java

```diff
@@ -124,13 +202,27 @@ public Optional<PropertyMetadata<?>> getConnectorSessionPropertyMetadata(Connect
         return Optional.ofNullable(properties.get(propertyName));
     }
 
+    private Map<String, PropertyMetadata<?>> getWorkerSessionProperties()
+    {
+        List<PropertyMetadata<?>> workerSessionPropertiesList = workerSessionPropertyProviders.values().stream()
+                .flatMap(manager -> manager.getSessionProperties().stream())
+                .collect(toImmutableList());
+        Map<String, PropertyMetadata<?>> workerSessionProperties = new ConcurrentHashMap<>();
+        workerSessionPropertiesList.forEach(sessionProperty -> {
+            requireNonNull(sessionProperty, "sessionProperty is null");
+            // TODO: Implement fail fast in case of duplicate entries.
```

Comment:
Can we just validate now?

Reply:
I was planning on validating the session properties using the prefix, and the support for prefix was supposed to be added in a follow-up PR (tracked here: https://github.com/prestodb/presto/issues/23828), so I can address this then.